solutions/dashboard/build/docker-compose.yml

```diff
@@ -40,3 +40,8 @@ services:
     build:
       context: ../tools/jenkins/
       dockerfile: ./Dockerfile
+  logstash:
+    image: logstash:$TAG
+    build:
+      context: ./logstash/
```

Comment:
@Sanket292001 , why are we not putting logstash related files in `tools` folder like other tools?

Reply:
@Sanket292001, as discussed, lets do tools folder management in next PR